### DISCUSSION TITLE
Inscription receiving address?

### DISCUSSION THREAD

**Discussion by 0xankylosaurus** [ordinals/ord#1754] [Q&A]
Hey Ordinals community, I have 2 questions about inscription receiving address:
1. Can all Taproot addresses receive an inscription?
2. If (1) is true, could one Taproot address receive multiple inscriptions and could the wallet associated with the address send the inscriptions to another wallet without any issue?

Thanks for your help in advance!

**Comment by reinamora137** [comment]
1. yes
2. yes - assuming you use ord wallet to send inscriptions and don't do ANY other spend transactions within that wallet. You can receive btc into that wallet. Sending things via bitcoin-cli (or even doing an RBF) can screw up your inscriptions. Possibly permanently.

I learned the hard way and now have at least 6 inscriptions I can't send out of the wallet (ie .sell) because of an RBF transaction I did in that wallet. Hopefully these things can be recovered in the future, but I haven't found any definitive answers on that. Perhaps some more insight from others here. 

Sparrow wallet has some good documentation on how to handle inscriptions and also recommends not sending them via their wallet.
